app/platform/metrics.go

```diff
@@ -32,20 +32,18 @@ type platformMetrics struct {
 
 	metricsImpl einterfaces.MetricsInterface
 
-	cfgFn func() *model.Config
+	cfgFn      func() *model.Config
+	listenAddr string
 }
 
 func (ps *PlatformService) metricsImpl() einterfaces.MetricsInterface {
```

Comment:
Are we using this method anywhere now? It seems we are directly accessing `ps.metricsIface`

Reply:
Yeah, makes sense. I rushed to push the fix w/o properly reviewing it 😓